src/Tests/Microsoft.Diagnostics.Monitoring.UnitTests/AuthenticationTests.cs

```diff
@@ -195,14 +192,14 @@ public async Task ApiKeyAuthenticationSchemeTest()
         public async Task NegotiateAuthenticationSchemeTest()
         {
             await using MonitorRunner toolRunner = new(_outputHelper);
-            await toolRunner.StartAsync(DefaultTimeout);
+            await toolRunner.StartAsync(TestTimeouts.StartProcess);
 
             // Create HttpClient and HttpClientHandler that uses the current
             // user's credentials from the test process. Since dotnet-monitor
             // is launched by the test process, the usage of these credentials
             // should authenticate correctly (except when elevated, which the
             // tool will deny authorization).
-            using HttpClient httpClient = await toolRunner.CreateHttpClientDefaultAddressAsync(_httpClientFactory, DefaultTimeout, ServiceProviderFixture.HttpClientName_DefaultCredentials);
+            using HttpClient httpClient = await toolRunner.CreateHttpClientDefaultAddressAsync(_httpClientFactory, TestTimeouts.HttpApi, ServiceProviderFixture.HttpClientName_DefaultCredentials);
```

Comment:
nit: Should we create overloads in the client that automatically pass the default timeout